tests/models/edx/test_base.py

```diff
@@ -0,0 +1,84 @@
+"""Tests for the base event model"""
+
+import pytest
+from pydantic.error_wrappers import ValidationError
+
+from tests.fixtures.edx.server import BaseEventFactory
+
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {"ip": "51.255.9.27"},
+        {"ip": "8.8.8.8"},
+        {"ip": ""},
+        {"event_source": "server"},
+        {"agent": ""},
+        {"host": ""},
+        {"referer": ""},
+        {"referer": "https://www.fun-mooc.fr/"},
+        {"referer": "https://www.fun-mooc.fr/event?event_type=page_close"},
+        {"context__user_id": ""},
+        {"context__user_id": None},
+        {"context__user_id": 213456},
+        {"context__org_id": "", "context__course_id": ""},
+        {"context__org_id": "org", "context__course_id": "course-v1:org+any+any"},
+    ],
+)
+def test_models_edx_base_event_with_valid_content(kwargs):
+    """Test that a valid base event does not raise ValidationError."""
```

Comment:
```suggestion
    """Test that a valid base event does not raise a ValidationError"""
```

Reply:
Thanks) Updated this but kept the last `.`